src/partner/modules/EventList/EventListContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import eventService from "../../services/EventService";
+import { IEvent } from "../../interfaces/Event";
+import EventList from "./EventList";
+
+interface IEventListContainerState {
+  events: IEvent[];
+  isLoading: boolean;
+}
+
+class EventListContainer extends React.Component<{}, IEventListContainerState> {
+  state = {
+    events: [],
+    isLoading: false
+  };
+
+  public componentDidMount() {
+    this.setState({ isLoading: true });
+    eventService.getAll().then(events => {
+      events.sort(function(a, b) {
```

Comment:
extract into a separate function

Reply:
Do you want the "sort function" as a service or some like that?